runtime.go

```diff
@@ -213,6 +214,10 @@ func (self *Runtime) Ticks() uint64 {
 	return self.ticks
 }
 
+func (self *Runtime) LimiterWaitCount() uint64 {
+	return self.limiterWaitCount
```

Comment:
@arahmanan the baas counterpart will be to log the LimiterWaitCount when the function execution finishes similar to how we log `Ticks()`. (This could also be a prometheus counter potentially, or perhaps I can piggyback on the Ticks log and include limiterWaitCount... we can discuss in the BAAS pr though)

Reply:
oh I like the idea of adding it to the ticks log! If we do that, adding a new prom metric won't be necessary.